caching-service/src/main/resources/application.yml

```diff
@@ -44,8 +44,8 @@ logging:
 
 eureka:
     client:
-        initialInstanceInfoReplicationIntervalSeconds: 1
```

Comment:
this one actually says `initialInstanceInfoReplicationIntervalSeconds` and we are switching it to `instanceInfoReplicationIntervalSeconds`. Is the first one correct?